src/Drupal/Driver/DrushDriver.php

```diff
@@ -120,7 +120,7 @@ public function isBootstrapped() {
    */
   public function userCreate(\stdClass $user) {
     $arguments = array(
-      $user->name,
+      sprintf('"%s"', $user->name),
```

Comment:
Doesn't this add `"` around the user name (meaning the name would be `"my user name"` rather than `my user name`)?